script.js

```diff
@@ -1,88 +1,69 @@
-    </div>
-    ` 
-
-});
-bookList.innerHTML = displ
-let removIds = [...document.querySelectorAll('.btn-display')].map(item => item.id)
-
-const newB = document.getElementById('bk')
-
-newB.addEventListener('submit', function(event){
+  {
+    titulo: '100 años de soledad',
+    author: 'Gabriel Garcia Marquez',
+  },
+  {
+    titulo: 'El tirano',
+    author: 'Massimo Manfredi',
+  },
+];
+
+function storage(ok) {
+  window.localStorage.setItem('localBooks', JSON.stringify(ok));
+}
 
-    event.preventDefault();
-    const reciTi = document.getElementById("recibe-ti").value
-    const reciAu = document.getElementById("recibe-au").value
+function getBooks() {
+  books = JSON.parse(window.localStorage.getItem('localBooks'));
+}
 
-    if(reciTi !== '' && reciAu !== ''){
-    let newBook = {
-        titulo: reciTi,
-        author: reciAu
-    }
-    books.unshift(newBook)
+getBooks();
 
-    let displ = ''
-    books.forEach((el, index) => {    
+function display() {
+  const bookList = document.querySelector('.books-display');
+  let displ = '';
+  books.forEach((el, index) => {
     displ += `
     <div class="oneBook" id="book${index}">
     <h4> ${el.titulo} </h4>
     <h4> ${el.author} </h4> 
     <button class="btn-display" id="${index}">remove</button> 
     </div>
-    `
-
-    });
-    bookList.innerHTML = displ
-
-    console.log(books)
+    `;
+  });
+  bookList.innerHTML = displ;
 }
-})
-
-/* document.querySelectorAll('.btn-display').forEach( (rem) => {
-    rem.addEventListener('click', (event) => {
-        console.log(rem)
-    })
 
-}) */
+// ---------ADD-------------
+
+const newB = document.getElementById('bk');
+newB.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const reciTi = document.getElementById('recibe-ti').value;
+  const reciAu = document.getElementById('recibe-au').value;
+
+  if (reciTi !== '' && reciAu !== '') {
+    const newBook = {
+      titulo: reciTi,
+      author: reciAu,
+    };
+    books.unshift(newBook);
+    storage(books);
+    display();
+  }
+});
 
-function selectBook(e){
-    
+// -------DELETE----------
+function deleteBook(evento) {
+  books = books.filter((el, index) => evento.target.id !== index.toString());
+  storage(books);
+  display();
 }
 
-document.addEventListener('click', e=>{
-    //let btnDelete = document.querySelector('.btn-display')
-    let removIds = [...document.querySelectorAll('.btn-display')].map(item => item.id)
-    //let numero = parseint(id, 10)
-    if(e.target.matches('.btn-display') ){
-        console.log('Hola');
-        //console.log(e.target.id);
-        removIds.filter((el, index) =>{
-            //console.log(index.toString(), 'a string')
-              e.target.id !== index.toString() })
-   ;
-    }
-
-    console.log(removIds);
-})
-
-
+document.addEventListener('click', (e) => {
+  if (e.target.matches('.btn-display')) {
+    deleteBook(e);
+  }
+});
 
-console.log(books);
+display();
```

Comment:
- [ ] Your display function is not working therefore The hard coded book items are not displayed on the initial page loading. PLease try to look at the error I got in the console and try to correct it 👇 👇 
![image](https://user-images.githubusercontent.com/90222110/182437424-792a59d7-6712-43e4-81e5-b752cc64db87.png)